proptest-state-machine/src/strategy.rs

```diff
@@ -575,7 +671,7 @@ mod test {
     ///
     /// This constant can be determined from the test
     /// `number_of_sequential_value_tree_simplifications`.
-    const SIMPLIFICATIONS: usize = 699;
+    const SIMPLIFICATIONS: usize = 700;
```

Comment:
this change was recently merged, so we'll need to resolve these two https://github.com/proptest-rs/proptest/pull/401/files#diff-4215d2d96283ff524e989ed16ad7059ed80dfc8e0c7c8d0467f549700fe7fdf2R570

Reply:
Hey, thanks for your patience. There's another thing that's changed in the last release that affects this PR - the `min_size` has been removed from `SequentialValueTree`. The changes in this PR are using it in `fn try_simplify` and `test_value_tree_initial_simplification`. I've started to rebase this PR, but I also have some other comments that I'd like to address

Reply:
I addressed this after rebase in https://github.com/proptest-rs/proptest/pull/388/commits/4c8696d9e470b6159711108ae43c616d6886c2fa